### ISSUE TITLE
Add-DbrServerToInventory falsely reports failures

### ISSUE THREAD

**Issue by ctrlbold** [sqlcollaborative/dbareports#63]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current behavior?**
Add-DbrServertoInventory falsely reports failures

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via https://jsfiddle.net**
Just downloaded most recent version

Get-SqlRegisteredServerName -sqlserver . | Add-DbrServerToInventory

etcetc
Something went wrong - The Beard is sad :-( . You can find the install log here ....

But everything went swimmingly.

**What is the expected behavior?**
YAYz

**Comment by SQLDBAWithABeard** [comment]
Yes, I spotted that on the last install and was going to add an issue :-)

You can't start the agent Job because the job is already running

I will add the code to perform that check and change the title of this issue

**Comment by SQLDBAWithABeard** [comment]
222bcd3  added
